### PR TITLE
Shooter

### PR DESCRIPTION
**Pull request by zbrotherton**
I have written some basic method headers and instance variables, as well as controls for a basic area within the subsystem. Shouldn't have much conflict with anything.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterSubsystem extends SubsystemBase {
+  private int aimMode; //0 is LOW, 1 is AUTO, 2 is LAUNCH, 3 is TARMAC
+
```
Comment (azvanderpas) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:14]: Consider adding method(s) to return the status of the flywheel and aiming for other subsystems/commands to use. 
e.g. the Hopper may want to know that the flywheel is spun up before sending a ball.

**Review by azvanderpas — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
```
Comment (sarango2158) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:6]: Suggestion: add TODO comments where we'll need to import and instantiate motor controllers

Question: Will you be doing this yourselves or using the MotorController class?

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  public void prime() {
+    //Check what aimMode is active, gets distance if AUTO, winds flywheel, adjusts hood correspondingly
+  }
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+  }
+
```
Comment (sarango2158) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:57]: Will you be publishing anything to the Smart Dashboard? What/when?

**Review by sarango2158 — APPROVED** [review]

**Review by sarango2158 — COMMENTED** [review]
You will need to merge in main again now that Intake has submitted their PR

**Pull request merged**